app/sim_test.go

```diff
@@ -662,8 +662,8 @@ func TestAppStateDeterminism(t *testing.T) {
 	config.OnOperation = false
 	config.AllInvariants = false
 
-	numSeeds := 3
-	numTimesToRunPerSeed := 5
+	numSeeds := 1
```

Comment:
hmm the idea is to run the non-determinism with 2 different seeds at least. Eg: one seed could run deterministic transactions but others not.
```suggestion
	numSeeds := 2
```